package.json

```diff
@@ -14,6 +14,7 @@
     "start:prod": "node dist/src/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "run-s migration:test && NODE_ENV=test jest",
+    "test-e2e": "run-s migration:test && jest --config ./test/jest-e2e.json --force-exit",
```

Comment:
just thinking if we really need the `jest-e2e.json` config file to achieve this as it could set a bad precedence that we have to create a config file for every test functionality type we wish to run separately, we could use regex here to select the e2e test files.